src/components/Personalization/turbine/initRuleComponentModules.js

```diff
@@ -24,10 +24,11 @@ import {
   appendHtml,
   prependHtml,
   insertHtmlAfter,
-  insertHtmlBefore
+  insertHtmlBefore,
+  click
 } from "../helper";
 
-export default collect => {
+export default (collect, storage) => {
```

Comment:
This parameter should probably be named `store` to be consistent.

Reply:
Done